### PR TITLE
SEO 4: Blog detail page

### PR DESCRIPTION
**Pull request by imrancs058**
- [x]  Title (<title>) of the blog detail page should be blog title.
- [x]  Facebook button on the header section of the blog should share the blog on Facebook.
- [x]  LinkedIn button on the header section of the blog should share the blog on LinkedIn.
- [x]  When we share the link of the blog, it should show a preview of the main image and text.

### CODE REVIEW

File: pages/blog/[slug].tsx [hunk 1]
```diff
@@ -23,79 +26,97 @@ import 'prismjs/plugins/normalize-whitespace/prism-normalize-whitespace'
...
+                  src={joinUsLinkIcons[0].icon.src}
+                  alt="icon"
+                  width={joinUsLinkIcons[0].width}
+                  height={joinUsLinkIcons[0].height}
+                  objectFit="cover"
+                />
+              </FacebookShareButton>
+              <LinkedinShareButton
+                url={
+                  'https://e963-39-40-250-233.ngrok-free.app/blog/annual-lunch-2023'
```
Comment (umairwaheed) [inline pages/blog/[slug].tsx:72]: Can you remove this hard-coded value?

Comment (umairwaheed) [reply pages/blog/[slug].tsx:72]:
```suggestion
`${process.env.NEXT_PUBLIC_BASEPATH}/${router.asPath}`
```

**Review by umairwaheed — CHANGES_REQUESTED** [review]
Remove hard-coded path.

File: pages/blog/[slug].tsx (hunk 1 above)
Comment (imrancs058) [reply pages/blog/[slug].tsx]: Done as suggested

**Comment by imrancs058** [comment]
> Remove hard-coded path.

Done as Suggested

**Review by umairwaheed — APPROVED** [review]

**Pull request merged**